src/system/settings/settings-keyboard.ts

```diff
@@ -135,7 +135,7 @@ export function setSettingKeyboard(setting: SettingKeyboard, value: number): boo
         if (globalScene.ui) {
           const cancelHandler = (success: boolean = false): boolean => {
             globalScene.ui.revertMode();
-            (globalScene.ui.getHandler() as KeyboardSettingsUiHandler).updateBindings();
+            (globalScene.ui.getCurrentHandler() as KeyboardSettingsUiHandler).updateBindings();
```

Comment:
NIT: Use generic param `<T>`

```suggestion
            globalScene.ui.getCurrentHandler<KeyboardSettingsUiHandler>().updateBindings();
```